examples/mundialis_example.py

```diff
@@ -31,8 +39,9 @@
 datacube = con.imagecollection("nc_spm_08.modis_lst.strds.MOD11B3.A2016306.h11v05.single_LST_Day_6km")
 datacube = datacube.filter_bbox(west=-80.5, south=32.5, east=-70.0, north=38.6, crs="EPSG:4326")
 datacube = datacube.filter_temporal(extent=["2016-01-01T00:00:00Z", "2016-03-10T23:59:59Z"])
+datacube = datacube.ndvi(nir="S2_4", red="S2_8A")
```

Comment:
the ndvi process [ndvi](https://processes.openeo.org/#ndvi) does not take nir and red as arguments
use datacube = datacube.ndvi()